tasks/conf/template-config.yml

```diff
@@ -28,6 +28,20 @@
   with_dict: "{{ nginx_http_template }}"
   when: nginx_http_template_enable
 
+- name: "(Setup: All NGINX) Ensure NGINX Proxy Cache Directories Exists"
```

Comment:
This looks a little bit messy as it is. Do you think you could clean it up a little bit?

Reply:
If I get it right, it looks messy because of multiple conditions? Now I removed them because `with_subelements` flag `skip_missing` do the same in this case.